Rakefile

```diff
@@ -14,12 +14,8 @@ end
 require 'cucumber'
 require 'cucumber/rake/task'
 
-def jruby?
```

Comment:
Backed this change out in favor of using cucumber's features/support/hooks.rb, which allows you to define annotations that can tag a scenario with code that executes before the test is run. In this case, I created two annotations that will instruct cucumber to skip the test. The result is functionally the same as what we had here, but the tests will be marked as skipped in the log output (before, the tests were not even read, so there was nothing in the cucumber run log about them).